README.md

```diff
@@ -58,4 +58,9 @@ Assuming you set a custom name to your SQL Server instance, you may invoke the i
 .\import.ps1 -server_name ".\MyCustomInstanceName"
 ```
 
+If you encounter the error `script cannot be loaded because running scripts is disabled on this system` follow these steps:
+	- Open PowerShell as Administrator and type: `Set-ExecutionPolicy RemoteSigned -Scope CurrentUser`
+	- Run the `Import.ps1` script again.
+	- After the import script completes, you can restrict script execution again by typing: `Set-ExecutionPolicy Restricted -Scope CurrentUser`
+
```

Comment:
Couple of things to note here:
  - The execution policy should be set to `Bypass` instead of `RemoteSigned`, since we don't sign nor need to sign the ps scripts in the project and we don't need to restore it to `Restricted` since anyway this is a dev machine that needs to run powershell scripts, assuming they want to work on this project
  - I just learned something new from your changes in regards to `-Scope CurrentUser`. This does not requires to run ps as admin, which makes sense, since we only affecting the current user. This is brilliant! because I learned now that other ps commands, such as `Install-Module` and `Install-PackageProvider` also supports setting the scope! so I'll update the scripts and docs and make the `config.cmd` in the main repo updating the execution policy, so that we simplify our docs. We should probably only this in the `Prerequisite` section of ko-db project in case one clones this project directly and now running the `config.cmd` via the main project. I would probably mentioned something like this in the prerequisites:
    - Being able to run powershell scripts. Note that if you're unable to run the scripts, it is because you need to allow powershell scripts to run on your system by setting the execution policy to bypass with the following powershell command: `Set-ExecutionPolicy Bypass -Scope CurrentUser`